NitroxPatcher/Patches/Persistent/EntityCell_AwakeAsync_Patch.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Reflection.Emit;
+using Harmony;
+using NitroxModel.Helper;
+using NitroxClient.MonoBehaviours;
+
+namespace NitroxPatcher.Patches.Persistent
+{
+    public class EntityCell_AwakeAsync_Patch : NitroxPatch
+    {
+        public static readonly Type TARGET_CLASS = typeof(EntityCell);
+
+        private static readonly FieldInfo current = getLoadAsyncEnumerableMethod().GetField("$current", BindingFlags.Instance | BindingFlags.NonPublic);
+
+        public static IEnumerable<CodeInstruction> Transpiler(MethodBase original, IEnumerable<CodeInstruction> instructions, ILGenerator generator)
+        {
+            List<CodeInstruction> codeInstructions = instructions.ToList();
+            yield return new CodeInstruction(OpCodes.Ldarg_0);
```

Comment:
Can you add some comments on what the IL is supposed to do? Ex: "Injecting field to set X so that Y happens."